test/test_images/hellovolume/hellovolume.go

```diff
@@ -44,6 +44,10 @@ func handler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// TODO: List cluster resources by using the bearertoken
+	// TOKEN=$(cat /base/token)
+	// curl -H “Authorization: Bearer $TOKEN” https://kubernetes/api/v1/ --insecure
```

Comment:
This doesn't need to live in here I believe? Could the test instead take the content from the response and try to do API requests?